web/app/css/breadcrumb-header.css

```diff
@@ -0,0 +1,17 @@
+@import 'styles.css';
+
+.ant-layout-header {
+  background-color: var(--header-black);
```

Comment:
opinion: I think it would look less weird if the header colour here and the sidebar were different. Like maybe a gray or green for this part? The mocks use `#26e99d` which I think doesn't look too bad. (Though it would be really really nice if we could also change our green/reds we use for the status dots to also match this.)

<img width="926" alt="screen shot 2018-09-07 at 4 18 38 pm" src="https://user-images.githubusercontent.com/549258/45247054-b6c72b00-b2b9-11e8-90e7-a413317f5f12.png">